serverless/functions/how-to/create-a-function.mdx

```diff
@@ -25,19 +25,17 @@ This page shows you how to deploy your first Scaleway Function. A function is a
 1. Click **Functions** in the **Serverless** section of the side menu. The functions page displays.
 2. Click the functions namespace you want to use to create your function. 
 3. Click **Create function**. The function creation wizard displays.
-    <Lightbox src="scaleway-create-a-function.webp" alt="" />
 4. Complete the following steps in the wizard: 
     - Enter your **function code**. 
-      - From the first drop-down menu, choose to either use the inline code editor to edit your code directly from your browser in the box provided, or to upload a `*.zip` archive.
-      - From the second drop-down menu, choose your desired runtime, e.g. `node16`, `node18`, `python37`, `python310` or `go118`.
-      - From the third drop-down menu, set the Handler for your function
+      - Set the Handler for your function.
+      - Choose your desired runtime and its version.
+      - Choose to either use the inline code editor to edit your code directly from your browser in the box provided, or to upload a `*.zip` archive.
     - Choose a **name** for your function, and optionally a **description**. The name must only contain alphanumeric characters.
     - Choose the **resources** to be allocated to your function at runtime. These define the performance characteristics of your function.
     - Set your **scaling** preferences, or leave them at default values. The Scaleway platform autoscales the number of available instances of your function to match the incoming load, depending on the settings you define here.
```

Comment:
```suggestion
    - Set your **scaling** preferences, or leave them at default values. The Scaleway platform autoscales the number of available Instances of your function to match the incoming load, depending on the settings you define here.
```

Reply:
Actually I think here we are talking about instances as a concept and not Instances as in the product, so it should be lower case? On the other hand, I don't know if "Handler" (above) should be capitalized - my instinct is not, but I see it's already like that in the existing doc! Maybe check with Bene, and we should add it to the wording handbook.

Reply:
Instance/instance is always tricky, I figured since it was within the scw console context that it should be capitalized but now that you're saying it you're probably right.